plugins/janus_sip.c

```diff
@@ -4357,10 +4357,11 @@ static void *janus_sip_handler(void *data) {
 				goto error;
 			json_t *action = json_object_get(root, "action");
 			const char *action_text = json_string_value(action);
-			if(strcasecmp(action_text, "start") && strcasecmp(action_text, "stop")) {
-				JANUS_LOG(LOG_ERR, "Invalid action (should be start|stop)\n");
+			if(strcasecmp(action_text, "start") && strcasecmp(action_text, "stop") &&
+				strcasecmp(action_text, "pause") && strcasecmp(action_text, "resume")) {
```

Comment:
This needs another tab of indentation, since it's a continuation of the `if` above and is confused with the action below.